libs/core/langchain_core/prompts/base.py

```diff
@@ -44,6 +44,8 @@ class BasePromptTemplate(
 
     input_variables: List[str]
     """A list of the names of the variables the prompt template expects."""
```

Comment:
Could you update the doc-string for input_variables to document how it should be interpreted?

Reply:
Is it better?

Reply:
Is it any better?